spec/requests/point_details_spec.rb

```diff
@@ -0,0 +1,301 @@
+require 'rails_helper'
+
+RSpec.describe "PointDetails", type: :request do
+  describe "#update" do
+    let(:point_detail) { create(:point_detail) }
+    let(:point_request) { point_detail.point_request }
+    let(:evaluation) { point_request.evaluation }
+    let(:group) { evaluation.group }
+    let(:principle) { point_detail.principle }
+    let(:point) { principle.max_per_member }
+    let(:point_detail_user) { point_request.user }
+    let(:arguments) do
+      ["/groups/#{group.id}/evaluations/#{evaluation.id}/pointdetails/update",
+       params: {
+           user_id:       point_detail_user.id,
+           principle_id:  principle.id,
+           evaluation_id: evaluation.id,
+           point:         point
+       },
+       headers: { "ACCEPT" => "application/js" }]
+    end
```

Comment:
I have the above mentioned feelings about the DRY-ness.

Reply:
In this case, I think it's not that confusing. The idea behind it that, you don't have to know how the arguments are put together, they are just set.